src/common/dpi_aware.cpp

```diff
@@ -4,7 +4,7 @@
 #include <ShellScalingApi.h>
 
 HRESULT DPIAware::GetScreenDPIForWindow(HWND hwnd, UINT &dpi_x, UINT &dpi_y) {
-  auto monitor_handle = MonitorFromWindow(hwnd, MONITOR_DEFAULTTONEAREST);
+  auto monitor_handle = MonitorFromWindow(hwnd, MONITOR_DEFAULTTOPRIMARY);
```

Comment:
With your changes, `GetScreenDPIForWindow()` is not called anymore by FanyzyZone, right?
If that is the case please leave `MONITOR_DEFAULTTONEAREST` because we are using it in an upcoming PowerToy that should not default to `MONITOR_DEFAULTTOPRIMARY`
Thanks.

Reply:
Sure. No problem. Will be fixed with the next commit